src/services/family-members.ts

```diff
@@ -0,0 +1,55 @@
+import { api, endpoints } from "./api";
+
+export type CreateFamilyMemberData = {
+  fullName: string;
+  relationship: string;
+  phoneNumber: string;
+  studentIds: number[];
+  socialName?: string;
+  gender?: string;
+  educationLevel?: string;
+  dateOfBirth: string;
+  socialPrograms?: string;
+  registrationNumber: string;
+  email?: string;
+  race?: string;
+  employmentStatus?: string;
+  nis?: string;
+};
+
+export type CreateAddressData = {
+  street: string;
+  neighborhood: string;
+  city: string;
+  state: string;
+  cep: string;
+  number: string;
+  complement?: string;
+};
+
+export type FamilyMemberResponse = {
+  id: string;
+};
+
+export async function createFamilyMember(data: CreateFamilyMemberData): Promise<FamilyMemberResponse> {
+  try {
+    console.log("Creating family member with data:", data);
+    const response = await api.post<FamilyMemberResponse>(endpoints.familyMembers.base, data);
+    return response.data;
+  } catch (error) {
+    console.error("API Error in createFamilyMember:", error);
+    throw error;
```

Comment:
Ótimo!
Seria legal ter um mock para testes de dev e cenários sem conexão ao banco (localmente).
Algo como:
```suggestion
    if (import.meta.env.PROD) {
      console.error("API Error in createFamilyMember:", error);
      throw error;
    } else {
      return 1;
    }
```